docs/pages/ai-assist.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Try out Teleport Assist.
+description: This tutorial will guide you through the steps needed to run Teleport Assist on your own infrastructure.
+---
+
+<Admonition type="tip" title="Preview">
+  Teleport Assist it currently in Preview.
+</Admonition>
+
+This guide will help you understand how to set up and use Teleport Assist, a new feature that enhances your remote access experience.
```

Comment:
Replace "enhances your remote access experience" with what it actually allows you to do, otherwise reads like fluff.